questionnaire-en.json

```diff
@@ -195,7 +186,7 @@
         },
         {
           "correctAnswer": "Yes",
-          "question": "Have the identified Program roles been properly staffed and has adequate funding provided?",
+          "question": "Have the identified Program roles been properly staffed and adequately funded?",

```

Comment:
Have or are?

This is a recurring question that popped up in my head when reading these changes – it seems to change most requirements to a finished past event. I assume that makes sense, but wanted to double-check, as to me as a non-native speaker this reads as it might be OK to tick off, if the requirements were met at some point in the past, but aren’t anymore.

(other then that, I really like all of the changes so far)

Reply:
@silverhook Good question - I personally would interpret it as being having been met and is currently met.  However, from a precise language point of view, this could be interpreted as having met the requirement in the past but not current.

Since we are nearing the release of the ISO standard, I'm going to go ahead and merge this as is.  We could open an issue to track the more general wording change for a future issue.

Reply:
I stuck with the original language form from the standard, so we should probably move review towards the next version of the spec itself?